source/reference/menus.rst

```diff
@@ -0,0 +1,107 @@
+===================
+ Menus - Reference
+===================
+
+.. seealso::
+
+   :doc:`Documentation on menus </customization/menus>`
+      Explains how menus work and what you can do
+
+
+File Format
+===========
+
+=============  ===========================================
+**Format**     JSON (with comments)
+
+**Extension**  ``.sublime-menu``
+
+**Name**       One out of the list of available menus.
+               See :ref:`menu-types` for the complete name list
+               and what menu they represent.
+
+**Location**   Any
+
+**Content**    A list of :ref:`menu-item-objects`.
```

Comment:
I am really unsure about that.

In other API docs, these "dictionaries" or "mappings" are usually referred to as "objects". It also makes sense from a javascript perspective, or JSON even, because that's what they are. And we need to differentiate this from the general "menu items" topic in the introductory page.